.github/workflows/documentation.yml

```diff
@@ -0,0 +1,115 @@
+        with:
+          node-version: 16
+          cache: yarn
+          registry-url: https://registry.npmjs.org/
+      - name: Install Dependencies
+        run: yarn --immutable
+      - name: Build Packages
+        run: yarn build
+      - name: Build Documentation
+        run: yarn docs
+      - name: Upload Documentation Artifacts
+        uses: actions/upload-artifact@v2
+        with:
+          name: docs
+          path: packages/*/docs/api.json
+      - name: Set Output
+        id: env
+        run: |
+          echo "::set-output name=NAME::${GITHUB_REF_NAME}"
+          echo "::set-output name=TYPE::${GITHUB_REF_TYPE}"
+          echo "::set-output name=SHA::${GITHUB_SHA}"
+
+  upload:
+    name: Upload Documentation
+    needs: build
+    strategy:
+      max-parallel: 1
+      fail-fast: false
+      matrix:
+        package: ['json', 'map', 'mongo', 'redis']
+    runs-on: ubuntu-latest
+    env:
+      NAME: ${{ needs.build.outputs.NAME }}
+      TYPE: ${{ needs.build.outputs.TYPE }}
+      SHA: ${{ needs.build.outputs.SHA }}
+    steps:
+      - name: Checkout Project
+        uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b # tag=v3
+      - name: Use Node.js v16
+        uses: actions/setup-node@eeb10cff27034e7acf239c5d29f62154018672fd # tag=v3
+        with:
+          node-version: 16
+          cache: yarn
+          registry-url: https://registry.npmjs.org/
+      - name: Install Dependencies
+        run: yarn --immutable
+      - name: Download Documentation Artifacts
+        uses: actions/download-artifact@fb598a63ae348fa914e94cd0ff38f362e927b741 # tag=v3
+        with:
+          name: docs
+          path: docs
+      - name: Checkout Documentation Project
+        uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b # tag=v3
+        with:
+          repository: 'josh-development/docs'
+          token: ${{ secrets.ACCESS_TOKEN }}
+          path: 'out'
+
+      - name: Extract Package and Semver
+        if: ${{ env.TYPE == 'tag' }}
+        id: extract-tag
+        uses: ./scripts/formatTag
+        with:
+          tag: ${{ env.NAME }}
+
+      - name: Move Documentation
+        if: ${{ env.TYPE == 'tag' }}
+        env:
+          PACKAGE: ${{ steps.extract-tag.outputs.package }}
+          SEMVER: ${{ steps.extract-tag.outputs.semver }}
+        run: |
+          mkdir -p out/${PACKAGE}
+          mv docs/${PACKAGE}/docs/api.json out/${PACKAGE}/${SEMVER}.json
+      - name: Move Documentation
+        if: ${{ env.TYPE == 'branch' }}
+        env:
+          PACKAGE: ${{ matrix.package }}
+        run: |
+          mkdir -p out/${PACKAGE}
+          mv docs/${PACKAGE}/docs/api.json out/${PACKAGE}/${NAME}.json
+
+      - name: Commit & Push
+        env:
+          PACKAGE: ${{ matrix.package }}
+        run: |
+          cd out
+          git config user.name github-actions[bot]
+          git config user.email 41898282+github-actions[bot]@users.noreply.github.com
```

Comment:
Is this email in ENV somewhere, seems weird to exactly specify the number

Reply:
That's the ID of the bot